ompi/runtime/ompi_mpi_init.c

```diff
@@ -124,11 +124,12 @@ const char ompi_version_string[] = OMPI_IDENT_STRING;
  * Global variables and symbols for the MPI layer
  */
 
-bool ompi_mpi_init_started = false;
-bool ompi_mpi_initialized = false;
-bool ompi_mpi_finalized = false;
-bool ompi_rte_initialized = false;
-int32_t ompi_mpi_finalize_started = false;
+opal_mutex_t ompi_mpi_bootstrap_mutex;
```

Comment:
@hjelmn @bosilca What is the Right way to initialize this mutex?  I notice that it's a class -- not a plain pthread mutex.  Should I use a simple pthread mutex instead, and then statically initialize it with PTHREAD_MUTEX_INITIALIZER?